### DISCUSSION TITLE
IP blocked on hosted weblate?

### DISCUSSION THREAD

**Discussion by softins** [WeblateOrg/weblate#12141] [Q&A] [answered]
I wanted to register for hosted weblate using my Github account, so I clicked the Register with Github button.

It didn't seem to do anything, so I tried a few times. That appeared to get my IP banned, and I can't now reach hosted.weblate.org

Do IP's automatically get unbanned after a time? If so, how long? If not, who do I ask to unban it?

Thanks!

**Comment by nijel** [comment, marked as answer]
A lot of terminated requests might cause temporary blocking, I think it is for one hour.

**Reply by softins** [reply]
Thanks, I'm back in now.
